javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -38,6 +38,7 @@
     private String indent = "    ";
     private String endOfLineCharacter = EOL;
     private Function<PrettyPrinterConfiguration, PrettyPrintVisitor> visitorFactory = PrettyPrintVisitor::new;
+    private int maxEnumConstantsToAlignHorizontally = 5;
```

Comment:
We could add a public static constant named `DEFAULT_MAX_ENUM_CONSTANTS_TO_ALIGN_HORIZONTALLY-

Reply:
I used to do that because it was standard advice from books and teachers, but I've noticed that in the case of a single use constant it is often clearer not to. In this case: the line says that the default value for `maxEnumConstantsToAlignHorizontally` is 5. Making a constant would only cause an unnecessary indirection in my opinion.

Reply:
It could be useful if we make it public, so that users could check what is default value and use it in their logic somehow

Reply:
Ah well, here you go then.